src/components/ContactCard/ContactForm/contactToFormValues.js

```diff
@@ -6,6 +6,22 @@ export const moveToHead = shouldBeHead => items =>
 
 const movePrimaryToHead = moveToHead(v => v.primary)
 
+/**
+ *
+ * @param {object|undefined} item - Contact attribute
```

Comment:
Est-ce qu'on est sûr que `item` peut être indéfini ? 🤔 
Si oui, tu peux écrire la jsDoc plutôt ainsi
```suggestion 
 * @param {object} [item] - Contact attribute
```

Reply:
modifié